docs/install/prerequisites.md

```diff
@@ -0,0 +1,40 @@
+# Prerequisites
+
+Before getting started with Percona Everest, we recommend that you:
+{.power-number}
+
+1. Install [AWS CLI](https://docs.aws.amazon.com/cli/latest/userguide/getting-started-install.html){:target="_blank"} for EKS or the [gcloud CLI](https://cloud.google.com/sdk/docs/install){:target="_blank"} for GKE.
+
```

Comment:
I wouldn't put this a prerequisite, this should only be part of the pages of how to create a kubernetes cluster, this isn't needed for everest itself

Reply:
Fixed